services/app-api/serverless.yml

```diff
@@ -253,10 +255,16 @@ functions:
       - arn:aws:lambda:us-east-1:901920570463:layer:aws-otel-nodejs-amd64-ver-1-18-1:1
 
 resources:
+  Conditions:
+    IsDevValProd: !Or
+      - !Equals ['${sls:stage}', 'main']
+      - !Equals ['${sls:stage}', 'val']
+      - !Equals ['${sls:stage}', 'prod']
   Resources:
     ApiGwWebAclAssociation:
       Type: AWS::WAFv2::WebACLAssociation
       DependsOn: 'ApiGatewayDeployment${sls:instanceId}'
+      Condition: IsDevValProd
```

Comment:
why are you modifying the ACL in this PR? Since you're checking the IP in code I didn't think there needs to be any change here

Reply:
Sorry that's a holdover from this [convo ](https://cmsgov.slack.com/archives/C014CRU197U/p1706289589824939)